soc/wch/ch32v/qingke_v4/vector.S

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2024 Michael Hope
+ * Copyright (c) 2025 MASSDRIVER EI (massdriver.space)
```

Comment:
Why change the copyright if the file is exactly the same?

Reply:
The file is changed on the first commit

Reply:
It has been moved to the first commit, please approve the PR.